libbeat/beat/beat.go

```diff
@@ -155,6 +156,10 @@ func (b *Beat) launch(bt Creator) error {
 		return err
 	}
 
+	if err := plugin.Initialize(); err != nil {
```

Comment:
Can we "hide" this behind a config option something like `plugins.enabled: false`? This make sure nothing happens by default and will make it easier to mark this experimential. Especially as it is not supported on all platforms and only go 1.8.